axonserver/src/main/vuejs/components/component/Processors.vue

```diff
@@ -366,7 +366,7 @@
             changeLoadBalancingStrategy(processor, strategy) {
                 console.log("strategy changed: " + strategy);
                 axios.put("v1/processors/" + encodeURIComponent(processor.name) +
-                                  "/autoloadbalance?context=" + encodeURIComponent(this.context) +
+                                  "/autoloadbalance?context=" + encodeURIComponent(processor.context) +
```

Comment:
there are some other references to this.context that need to be replaced by processor.context.